concent_api/utils/logging.py

```diff
@@ -325,6 +334,42 @@ def log_operation_validation_failed(
     )
 
 
+@replace_element_to_unavailable_instead_of_none
+def log_message_received_in_endpoint(
+    logger,
+    application_and_endpoint: str,
+    message_type: str,
+    client_public_key: bytes,
+    content_type: Optional[str] = None,
+    task_id: Optional[str] = None,
+    subtask_id: Optional[str] = None
+
+):
+
+    logger.info(
+        f'A message has been received in `{application_and_endpoint}/` '
+        f'Message type: {message_type} '
+        f'{"TASK_ID:" if task_id is not None else ""} {task_id if task_id is not None else ""}'
+        f'{" SUBTASK_ID:" if subtask_id is not None else ""} {subtask_id if subtask_id is not None else ""}'
+        f'CLIENT_PUBLIC_KEY: {client_public_key} '
+        f'{"Content type:" if content_type is not None else ""} {content_type if content_type is not None else ""} '
```

Comment:
I don't get that format, why did You change it from:
```
f'{"Content type:" content_type}'
```
?

Reply:
Okey, but why duplicate printing variable with `f'{} {}'` instead of `f'{}'`?

Reply:
I did it, because it is some logic in first `{}` and second `{}`. Now it's duplicated like `f'{} {}'`, but the logic for each `{}` is separated and it's much easier to read and understand this then if it would be in one massive `{}`, i think.